pkg/sysfs/system.go

```diff
@@ -152,6 +152,7 @@ type CPU interface {
 	ThreadCPUSet() cpuset.CPUSet
 	BaseFrequency() uint64
 	FrequencyRange() CPUFreq
+	EPP() EPP
```

Comment:
Just a note, not something immediate: it might be necessary/useful later to add a `System.EPPCPUSet(EPP) cpuset.CPUSet` or something to that effect, for better configurability for the CPU allocator by the caller.

Reply:
Mm, certainly, that is a good idea as EPP has a limited pre-defined set of values. I'll keep that in mind as this needs to be revisited when implementing more extensive SST support